packages/actor-query-operation-distinct-hash/package.json

```diff
@@ -36,7 +36,7 @@
   ],
   "dependencies": {
     "@comunica/actor-abstract-bindings-hash": "^1.13.0",
-    "json-stable-stringify": "^1.0.1"
+    "canonicalize": "^1.0.1"
```

Comment:
It looks like this dependency can even be removed here, since it appears it's not being used directly in this package.